snuba/clickhouse/http.py

```diff
@@ -233,7 +256,7 @@ def join(self, timeout: Optional[float] = None) -> None:
                 try:
                     if row is not None:
                         sentry_sdk.set_context(
-                            "snuba_errored_row", {"data": self.__rows[row]}
+                            "snuba_errored_row", {"data": self.__debug_buffer[row]}
```

Comment:
Since the debug buffer doesn't necessarily contain all the rows, couldn't this throw an IndexError?

Reply:
yeah but the indexerror is caught down